pkgs/development/libraries/openldap/default.nix

```diff
@@ -34,6 +34,13 @@ stdenv.mkDerivation rec {
     "devdoc"
   ];
 
+  postPatch =
+    # flaky/nondeterministic tests; see https://github.com/NixOS/nixpkgs/pull/209870#issuecomment-1474492037
+    if stdenv.buildPlatform.isAarch64 then ''
+      rm tests/scripts/test063-delta-multiprovider
+      rm tests/scripts/test069-delta-multiprovider-starttls
+    '' else null;
```

Comment:
Since we got some lib.optionalString treewides changes in recently, we should probably not straight away create a new candidate which might trigger a stdenv rebuild.

Reply:
Thanks, I will drop the last two commits before merging.

https://github.com/NixOS/nixpkgs/pull/209870#issuecomment-1487625275